addons/html_editor/static/src/editor/font/font_plugin.js

```diff
@@ -242,6 +244,24 @@ export class FontPlugin extends Plugin {
             return true;
         }
     }
+
+    handleDeleteBackward({ startContainer, endContainer }) {
+        const handledTags = [...headingTags, "PRE", "BLOCKQUOTE"];
+        const selector = handledTags.join(", ");
```

Comment:
you can move the selector just at line 87:
```js
        const handledElemSelector = [...headingTags, "PRE", "BLOCKQUOTE"].join(", ");
```
so it is evaluated only once

Reply:
Done.